en/docs/develop/deploying-the-sample-app.md

```diff
@@ -4,87 +4,364 @@ This topic provides instructions on how to download and deploy the
+
+        3.  **Enable Attribute Profile**
+
+        4.  **Include Attributes in the Response Always**  
+        
+        5.  **Enable Signature Validation in Authentication Requests and Logout Requests**
+            
+
+    ![edit-service-provider](../assets/img/tutorials/edit-service-provider-configs.png)
+    
+    !!! tip
+        For more information on the advanced configurations
+        refer, [Configuring OAuth2-OpenID Connect Single-Sign-On](../../learn/configuring-saml2-web-single-sign-on)
+
+5.  Click **Register** to save the changes.  
+    Now you are sent back to the Service Providers page.
+
+
+##Deploying playground2 webapp
+
+### Download the samples
+
+To be able to deploy a sample of Identity Server, you need to download
+it onto your machine first.
+
+Follow the instructions below to download a sample from GitHub.
+
+1. Navigate to [WSO2 Identity Server Samples](https://github.com/wso2/samples-is/releases).
+2. Download the `playground2.war` file from the latest release assets.
+
+### Deploy the sample web app
+
+Deploy this sample web app on a web container.
+
+1.  Copy the .war file into the `webapps`
+    folder. For example,
+    `<TOMCAT_HOME>/apache-tomcat-<version>/webapps`
+    .
+2.  Start the Tomcat server.
+
+To check the sample application, navigate to
+`http://<TOMCAT_HOST>:<TOMCAT_PORT>/playground2/oauth2.jsp`
+on your browser.
+
+For example,
+`http://localhost:8080/playground2/oauth2.jsp`
+
+    
+### Configuring Service Provider
+
+The next step is to configure the service provider.
+
+1.  Return to the WSO2 IS management console.
+
+2.  Navigate to **Main** tab -> **Identity** -> **Service Providers** -> **Add**.
 
-    Access the samples by navigating to the
-    `            is-samples/modules/samples           ` directory.
+3.  Enter **playground2** in the **Service Provider Name** text box,
+    and click **Register**.
+        ![]( ../assets/img/103329986/103329999.png) 
+    
+4.  Expand the **Inbound Authentication Configuration** section and then
+    the **OAuth/OpenID Connect Configuration** and click
+    **Configure.**   
+    
+5.  Fill in the form that appears. For the Allowed Grant Types, you can disable the ones you do not require or block.
+    
+    ![configure-oauth-oidc](../assets/img/tutorials/configure-oauth-oidc.png)
+        
+    !!! note
+    
+        The grant type highlighted below is a **custom** grant type. This
+        will only appear on the UI if you have [configured the JWT grant
+        type](../../develop/jwt-grant-type-for-oauth2). The value specified as the `name`
+        of the `oauth.custom_grant_type` in the `deployment.toml` file when
+        creating the custom grant type is the value that will appear on the
+        UI. For more information on writing a custom grant type, see
+        [Writing a Custom OAuth 2.0 Grant
+        Type](../../learn/writing-a-custom-oauth-2.0-grant-type).
+        
+6.  Fill the `Callback Url` field. 
```

Comment:
Let's provide exact sample URL, update the screenshot also with correct configured values